doc/source/user_guide/plots_arguments_guide.md

```diff
@@ -304,6 +304,24 @@ Matplotlib {doc}`mpl:tutorials/colors/colors` tutorial.
 Module {mod}`bokeh.colors`
 :::
 
+(common_rope)=
+## `rope`
+
+A list with lower and upper values of the Region Of Practical Equivalence. If a list with one interval
+only is provided, the `ROPE` will be displayed across the `y-axis`. If more than one interval is
+provided the length of the list should match the number of variables.
```

Comment:
I think this doesn't match the actual behaviour nor the example below. From the example below it looks like the recommended usage is a dictionary of list/tuple where at the same time, each element in those list/tuples is a dictionary too. 

The first dictionary has variable names as keys, and list/tuple as values, whereas the 2nd dictionaries have coord names as keys and coord values as values for those keys and also need a `rope` key with values a (float, float) tuple which defines the rope interval for that variable/subvariable.

We should also add a single list and a list with same length of variable names (which will also serve to double check the description is correct, which might not be the case)

Reply:
I am not fully following you.